dss/api/search.py

```diff
@@ -31,24 +31,23 @@ def post(json_request_body: dict,
     es_query = json_request_body['es_query']
     per_page = PerPageBounds.check(per_page)
 
-    if replica is None:
-        replica = "aws"
+    replica_enum = Replica[replica] if replica else Replica.aws
```

Comment:
Strictly speaking, `if replica` is not the same as `if replica is None`

Reply:
✅